### PR TITLE
[HTML- CSS-Responsive] Popup

### LINKED ISSUE #29: [HTML- CSS-Responsive] Popup
-Design
type-drink-popup1
type-drink-popup2

### PR DESCRIPTION
**Pull request by farahalzard**
Relates #29

### CODE REVIEW

**Review by IsraaHamdi — CHANGES_REQUESTED** [review]
In css file using grouping for selectors have same styles

**Comment by farahalzard** [comment]
> In css file using grouping for selectors have same styles
Done

**Review by IsraaHamdi — APPROVED** [review]

File: otherPages/type/typeDrink.html [hunk 1]
```diff
@@ -42,6 +42,35 @@
             
     </section>
     <!-- End Cards Section -->
+    <!-- Recipe popup -->
+    <div class="overlay">
+        <div class="pop-up"> 
+            <div id="cont">
+            <i class="fas fa-times-circle close"></i>
```
Comment (yaqootturman) [inline otherPages/type/typeDrink.html:49]: add the functionality of it

File: otherPages/type/typeFood.html [hunk 2]
```diff
@@ -44,6 +44,35 @@
         
     </section>
     <!-- End Cards Section -->
+    <!-- Recipe popup -->
+    <div class="overlay">
+        <div class="pop-up">
```
Comment (yaqootturman) [inline otherPages/type/typeFood.html:49]: I'm good with vertical scroll but horizontal one is messy, ask your team for suggestions

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: otherPages/type/typeDrink.html (hunk 1 above)
Comment (farahalzard) [reply otherPages/type/typeDrink.html:49]: It's a container for the head of the pop-up, I changed the id name

**Review by muhannaAtalla20 — APPROVED** [review]

**Review by yaqootturman — APPROVED** [review]

**Pull request merged**